server/src/test/java/org/candlepin/cache/StatusCacheTest.java

```diff
@@ -46,8 +47,8 @@ public void setAndGet() {
         StatusCache cache = new StatusCache();
 
         StatusDTO status = new StatusDTO()
-            .setResult(true)
-            .setTimeUTC(new Date());
+            .result(true)
+            .timeUTC(new Date().toInstant().atOffset(ZoneOffset.UTC));
```

Comment:
Same for all the changes in this file, just use `OffsetDateTime.now()` directly.

Reply:
Done.